examples/porting/lv_port_lcd_stm32_template.c

```diff
@@ -0,0 +1,183 @@
+/**
+ * @file lv_port_lcd_stm32_template.c
+ *
+ * Example implementation of the LVGL LCD display drivers on the STM32 platform
+ */
+
+/*Copy this file as "lv_port_disp.c" and set this value to "1" to enable content*/
+#if 0
+
+/*********************
+ *      INCLUDES
+ *********************/
+/* Include STM32Cube files here, e.g.:
+#include "stm32f7xx_hal.h"
+*/
+
+#include "lv_port_disp.h"
+#include "./src/drivers/display/st7789/lv_st7789.h"
+
+/*********************
+ *      DEFINES
+ *********************/
+#ifndef MY_DISP_HOR_RES
+    #warning Please define or replace the macro MY_DISP_HOR_RES with the actual screen width, default value 320 is used for now.
+    #define MY_DISP_HOR_RES    320
+#endif
+
+#ifndef MY_DISP_VER_RES
+    #warning Please define or replace the macro MY_DISP_HOR_RES with the actual screen height, default value 240 is used for now.
```

Comment:
Typo:  `MY_DISP_HOR_RES`  ->  `MY_DISP_VER_RES`

Reply:
Good catch! Actually I copied this from lv_port_disp_template.c. I fixed it in both places.